src/AvtaleSide/steg/GodkjenningSteg/Oppsummering/OppsummeringMentor/MentorIkon.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const MentorIkon = () => (
```

Comment:
Ikke ha SVG som  en TSX-fil. Ha den i SVG

Reply:
Den ligger vel forøvrig som svg også, og brukes på riktig måte, så denne svg-komponenten er vel heller ikke i bruk nå.